src/FixedProductMarketMakerMapping.ts

```diff
@@ -79,10 +79,8 @@ function recordTrade(fpmm: FixedProductMarketMaker,
     fpmmTransaction.user = traderAddress;
     fpmmTransaction.fpmmType = FPMM_TYPE_TRADE;
     fpmmTransaction.transactionType = tradeType;
-    fpmmTransaction.collateralAmount = collateralAmount;
-    fpmmTransaction.collateralAmountUSD = collateralAmountUSD;
-    fpmmTransaction.collateralTokenAddress = fpmm.collateralToken;
```

Comment:
@astro514 we should keep the collateralTokenAddress i need it for displaying what currency it is in the front-end

Reply:
@Mi-Lan  you can get it from the fpmm
collateralTokenAddress is the same for all transactions of a pool(fpmm)